app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self. clean_mark = clean_mark
```

Comment:
There is an extra space after `self.` in `self. clean_mark`. It should be `self.clean_mark` for consistency and to avoid potential errors.